R/calcIEA_WEO_2021.R

```diff
@@ -2,7 +2,9 @@
 #'
 #' @md
 #' @return A [`magpie`][magclass::magclass] object.
-#' @param aggregate Boolean indicating whether output data aggregation should be performed or not
+#' @param subtype Either "global" or "region". On global level, the source offers
+#' more variables than on regional level, but the data should not be used on sub-
+#' global level due to its coarse disaggregation.
```

Comment:
That is not a `subtype` argument as used throughout the madrat system.  This is a switch, that does some additional computations when it is set to `'global'`, and nothing otherwise.

Reply:
It also determines the subtype to be used for `read_IEA_WEO_2021` (which is the more important effect of the parameter)

Reply:
Ah, ok.  Getting passed on to `convertIEA_WEO_2021()`.